chat-widget/src/components/postchatsurveypanestateful/PostChatSurveyPaneStateful.tsx

```diff
@@ -34,7 +34,8 @@ export const PostChatSurveyPaneStateful = (props: IPostChatSurveyPaneStatefulPro
     let surveyInviteLink = "";
     const surveyMode = (state?.appStates?.selectedSurveyMode === PostChatSurveyMode.Embed);
 
-    if (state?.appStates?.conversationEndedBy === ConversationEndEntity.Bot && state.domainStates.postChatContext.botSurveyInviteLink) {
+    if (state.domainStates.postChatContext.botSurveyInviteLink && // Bot survey enabled
```

Comment:
Do we need to check if the conversation is ended/inactive ?

Reply:
The PostChatSurveyPaneStateful will only show up when the conversation is ended.